OPNsense/opnsense/ingest/parser.yml

```
@@ -0,0 +1,92 @@
+          PF_CARP: '%{DATA:hoplimit},%{DATA:vhid},%{DATA:version},%{DATA:advskew},%{DATA:advbase}'
+          PF_ICMP: 'datalenght=%{DATA:icmp_datalenght}'
+
+  - name: set_ecs_fields
+    stages:
+      set_ecs_fields:
+        actions:
+          - set:
+              source.ip: "{{parsed_event.message.src}}"
+              destination.ip: "{{parsed_event.message.dst}}"
+              rule.id: "{{parsed_event.message.rulenr}}"
+              openbsd.pf.rule.subrulenr: "{{parsed_event.message.subrulenr}}"
+              rule.ruleset: "{{parsed_event.message.anchorname}}"
+              openbsd.pf.event.tracker.id: "{{parsed_event.message.label}}"
+              event.reason: "{{parsed_event.message.reason}}"
+              event.action: "{{parsed_event.message.action}}"
+              network.direction: "{{parsed_event.message.dir}}"
+              openbsd.pf.routing.class: "{{parsed_event.message.tos}}"
+              openbsd.pf.routing.hoplimit: "{{parsed_event.message.hoplimit}}"
+              openbsd.pf.routing.offset: "{{parsed_event.message.offset}}"
+              openbsd.pf.routing.flags: "{{parsed_event.message.flags}}"
+              network.network.iana_number: "{{parsed_event.message.protonum}}"
+              network.transport: "{{parsed_event.message.protoname}}"
+              network.bytes: "{{parsed_event.message.length}}"
+              source.port: "{{parsed_event.message.srcport}}"
+              destination.port: "{{parsed_event.message.dstport}}"
+              openbsd.pf.transport.bytes: "{{parsed_event.message.datalen}}"
+              openbsd.pf.transport.tcp_flags: "{{parsed_event.message.tcp_flags}}"
+              openbsd.pf.transport.sequence_number: "{{parsed_event.message.seq}}"
+              openbsd.pf.transport.ack: "{{parsed_event.message.ack}}"
+              openbsd.pf.transport.window_size: "{{parsed_event.message.window}}"
+              openbsd.pf.transport.urgency: "{{parsed_event.message.urg}}"
+              openbsd.pf.transport.options: "{{parsed_event.message.option}}"
+              event.classification: "{{parsed_event.message.class}}"
+              openbsd.pf.carp.vhid: "{{parsed_event.message.vhid}}"
+              openbsd.pf.carp.version: "{{parsed_event.message.version}}"
+              openbsd.pf.icmp.advskew: "{{parsed_event.message.advskew}}"
+              openbsd.pf.icmp.advbase: "{{parsed_event.message.advbase}}"
+              opnsense.datalength: "{{parsed_event.message.datalenght}}"
+              openbsd.pf.routing.flow: "{{parsed_event.message.flow}}"
+
+          - set:
+              event.kind: 'event'
+              event.category: ['network']
+              event.type: ['connection']
+
+          - translate:
+              dictionary:
+                "blocked": ["denied"]
+                "dropped": ["denied"]
+                "quarantined": ["denied"]
+                "pass": ["allowed"]
+                "allow": ["allowed"]
+                "detect": ["allowed"]
```

Are you sure for the values on the left? I only remember `pass` and ` block` for packet filter.
```suggestion
                "block": ["denied"]]
                "pass": ["allowed"]
```